includes/class-wc-ajax.php

```diff
@@ -1300,6 +1299,8 @@ public static function remove_order_item() {
 			$order->calculate_taxes( $calculate_tax_args );
 			$order->calculate_totals( false );
 
+			do_action( 'woocommerce_ajax_order_items_removed', $item_id, $item, $changed_stock, $order );
```

Comment:
Please include a docblock for this hook. You can find instructions about it in https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#4-hooks-actions-and-filters

Reply:
Added.